### PR TITLE
memory leaks

### PR DESCRIPTION
**Pull request by tsaizhenling**
fixed memory leaks. SVGKit is till leaking. disable global cache reveals more leaks

### CODE REVIEW

File: Source/Parsers/SVGKParser.m [hunk 1]
```diff
@@ -83,7 +83,10 @@ - (void)dealloc {
 	self.source = nil;
 	[_storedChars release];
 	[_stackOfParserExtensions release];
+    [_parentOfCurrentNode release];
```
Comment (adamgit) [inline Source/Parsers/SVGKParser.m]: This is never retained - should not be released?

File: Source/SVGKImage.h [hunk 2]
```diff
@@ -46,7 +46,7 @@
 #import "SVGKSource.h"
 #import "SVGKParseResult.h"
 
-#define ENABLE_GLOBAL_IMAGE_CACHE_FOR_SVGKIMAGE_IMAGE_NAMED 1 // if ENABLED, then ALL instances created with imageNamed: are shared, and are NEVER RELEASED
```
Comment (adamgit) [inline Source/SVGKImage.h:49]: I understand you might not want to use this feature - that's why it's a #define (set it to 0 to compile it out).

But, for the main project, we need this turned ON by default, sorry. Can't merge this :(

**Comment by adamgit** [comment]
Good work, it's 99% great - but: I think you have 1 too many releases, and we have to leave the #define ON by default.

Please check those two items (I've commented in-line) and update, then I can merge this

Thanks!

File: Source/SVGKImage.h (hunk 2 above)
Comment (adamgit) [inline Source/SVGKImage.h:49]: Sorry, to be clear: it's a good idea to turn this off for testing - but I can't merge it, and Github won't let me "pick and choose" :(

File: Source/Parsers/SVGKParser.m (hunk 1 above)
Comment (tsaizhenling) [inline Source/Parsers/SVGKParser.m]: yes you are right

File: Source/SVGKImage.h (hunk 2 above)
Comment (tsaizhenling) [inline Source/SVGKImage.h:49]: done

**Comment by tsaizhenling** [comment]
Screen Shot 2013-03-02 at 12 15 25 AM
 left with these memory leaks. I can't quite figure out how to fix these :(

**Pull request merged**